app/cdap/components/ConnectionList/index.tsx

```diff
@@ -205,26 +208,26 @@ export default function ConnectionList() {
     <Box data-testid="data-sets-parent" className={classes.connectionsListContainer}>
       <SubHeader />
       <SelectDatasetWrapper>
-        {dataForTabs.map((each, index) => {
-          const connectionIdRequired = each.data.filter((el) => el.connectionId);
+        {dataForTabs.map((eachTab, tabIndex) => {
```

Comment:
always check whether a variable is not null and array and has length when u want to use array methods

Reply:
Resolved